### PR TITLE
Run the cs fixer for green ci

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

**Review by kesselb — APPROVED** [review]

**Review by kesselb — CHANGES_REQUESTED** [review]
Wait ;) I was to quick.

File: lib/private/Files/ObjectStore/Swift.php [hunk 1]
```diff
@@ -26,17 +26,12 @@
...
-use GuzzleHttp\Exception\ConnectException;
-use GuzzleHttp\Exception\RequestException;
 use GuzzleHttp\Exception\BadResponseException;
-use GuzzleHttp\HandlerStack;
 use function GuzzleHttp\Psr7\stream_for;
 use Icewind\Streams\RetryWrapper;
 use OCP\Files\NotFoundException;
 use OCP\Files\ObjectStore\IObjectStore;
 use OCP\Files\StorageAuthException;
-use OpenStack\Common\Error\BadResponseError;
```
Comment (kesselb) [inline lib/private/Files/ObjectStore/Swift.php:39]: Is that intended? To swap `OpenStack\Common\Error\BadResponseError` with `GuzzleHttp\Exception\BadResponseException`.

Comment (kesselb) [reply lib/private/Files/ObjectStore/Swift.php:39]: whoops :see_no_evil:

**Review by kesselb — APPROVED** [review]

File: lib/private/Files/ObjectStore/Swift.php (hunk 1 above)
Comment (MorrisJobke) [reply lib/private/Files/ObjectStore/Swift.php:39]: ...Exception vs ...Error

Comment (skjnldsv) [reply lib/private/Files/ObjectStore/Swift.php:39]: They're different?
BadResponseError/BadResponseException

**Review by ChristophWurst — APPROVED** [review]
:green_heart:

**Review by gary-kim — APPROVED** [review]

**Pull request merged**